### PR TITLE
chore(docs): migrate back to global installs

### LINKED ISSUE #12195: Revert instructing people to use npx and npm scripts
We recently started suggesting people use `npx` and `npm scripts` more which I think was a mistake.

There are some cli commands that we use constantly — `git`, `npm`, `ls`, `cd`, etc. These are all global commands that we want accessible at all times.

People often suggest using npm scripts for a few reasons:

- it ensures that you're using the locally installed version (perhaps you want to use a different version than a globally installed version)
- the perception that installing more globals clutters things

Neither apply to Gatsby. The Gatsby CLI version doesn't matter — when you run `gatsby develop` or `gatsby build` or any of the other site commands, it just looks for the locally installed gatsby package (and fails if it can't find one) and runs the command for you.

`npx` is an interesting idea for simplifying using seldomly run global commands as it skips the installation step for using a global + ensures that the global is always up to date.

But Gatsby is _not_ a seldomly used command — people use Gatsby all day long and it's very reasonable to expect that they'll install the gatsby-cli package once.

Also because people do type Gatsby commands so often, it's not very pleasant to need to type 4 extra characters every time. Imagine if you had to type `npx npm` or `npx git` or `npx ls`, etc.

Things to revert:

- [ ] GitHub issue templates
- [ ] getting started doc
- [ ] recipes
- [ ] tutorial
- [x] other random places in the docs where things where switched over

### PR DESCRIPTION
**Pull request by ryancharris**
## Description
This PR reverts all references to `npx` in the codebase that occur outside of the `docs/tutorial/` subdirectory.

A follow up PR from @DSchau will address the references in the tutorial docs.

## Related Issues
- Fixes #12195 
- Reverts some of the work done as part of [#11556](https://github.com/gatsbyjs/gatsby/pull/11556)

### CODE REVIEW

File: docs/docs/gatsby-cli.md [hunk 1]
```diff
@@ -28,15 +28,11 @@ Now we have the `develop` script available to be used which will use our package
 
 ## How to use
 
-By using [npx](https://www.npmjs.com/package/npx) you can utilize the Gatsby command line tool without installing it globally. This lets you scaffold a project with just one command!
-
-For example you can run `npx gatsby new your-site-name` to generate a new Gatsby site.
-
-Run `npx gatsby --help` for full help.
+Run `gatsby --help` for full help.
```
Comment (DSchau) [inline docs/docs/gatsby-cli.md]: This should reference globally installing it first and foremost!

File: packages/gatsby-cli/README.md [hunk 2]
```diff
@@ -23,15 +23,13 @@ We can now use the `develop` script tied to our app's version of Gatsby, rather
 
 ## How to use
 
-Rather than installing the Gatsby CLI tool locally you can run it using [npx](https://www.npmjs.com/package/npx)
-
-To see the commands available to run `npx gatsby --help`.
+Run `gatsby --help` for full help.
```
Comment (DSchau) [inline packages/gatsby-cli/README.md]: Same here, e.g. reference that it needs to be globally installed and _how_ to globally install.

File: docs/docs/gatsby-cli.md (hunk 1 above)
Comment (ryancharris) [reply docs/docs/gatsby-cli.md]: @DSchau You got it. Will update 🙌

File: packages/gatsby-cli/README.md (hunk 2 above)
Comment (ryancharris) [reply packages/gatsby-cli/README.md]: Same here

**Comment by ryancharris** [comment]
@DSchau Hit your notes in 3163c74

**Pull request merged**